src/pages/index.tsx

```diff
@@ -1,30 +1,166 @@
-import React from 'react';
-import clsx from 'clsx';
+import React, { useState, useEffect } from 'react';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import Layout from '@theme/Layout';
-import HomepageFeatures from '@site/src/components/HomepageFeatures';
+import { IoOpenOutline } from "react-icons/io5";
+import ReferenceCard from '@site/src/components/ReferenceCard';
 
 import styles from './index.module.css';
 
-function HomepageHeader() {
-  const {siteConfig} = useDocusaurusContext();
+function RotatingHeader() {
+  const { siteConfig } = useDocusaurusContext();
+  const actions = ['Build faster with', 'Integrate easier on', 'Cache data with', 'Publish messages using'];
+  const [index, setIndex] = useState(0);
+  const [fade, setFade] = useState(true);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setFade(false);
+      setTimeout(() => {
+        setIndex((prevIndex) => (prevIndex + 1) % actions.length);
+        setFade(true);
+      }, 500);
+    }, 5000);
+    return () => clearInterval(interval);
+  }, []);
+
   return (
-    <header className={clsx('hero hero--primary', styles.heroBanner)}>
-      <div className="container">
-        <h1 className="hero__title">{siteConfig.title}</h1>
-        <p className="hero__subtitle">{siteConfig.tagline}</p>
-      </div>
-    </header>
+    <div className={styles.headerSection}>
+      <h1 className={`${styles.rotatingHeader} ${fade ? styles.fadeIn : styles.fadeOut} ${styles.header}`}>
+        {actions[index]}
+      </h1>
+      <h1 className={styles.header}>the Momento Platform</h1>
+      <p className={styles.headerDescription}>Welcome to the Momento developer docs. Discover all the resources you need to help you get the most out of our products.</p>
+    </div>
   );
 }
 
 export default function Home(): JSX.Element {
   return (
     <Layout
       description="Description will go into a meta tag in <head />">
-      <HomepageHeader />
-      <main>
-        <HomepageFeatures />
+      <main className={styles.content}>
+        <div className={styles.contentContainer}>
+          <RotatingHeader />
+          <div className={styles.desktopOnly}>
+            <div className={styles.columns}>
+              <div>
+                <h2>Cache</h2>
+                <ul className={styles.columnList}>
+                  <li><a href="/cache/getting-started">Getting started</a></li>
+                  <li><a href="/cache/develop/api-reference">API reference</a></li>
+                  <li><a href="/cache/develop/guides/working-with-files-cache">How to cache files</a></li>
+                  <li><a href="/cache/how-to/rate-limiter">Build a rate-limiter</a></li>
```

Comment:
Should we have a more generic use-case on the landing page such as `read-aside caching`?